python/lsst/pipe/base/argumentParser.py

```diff
@@ -732,19 +724,28 @@ def __call__(self, parser, namespace, values, option_string):
         @param[in] parser       argument parser (instance of ArgumentParser)
         @param[in] namespace    parsed command (an instance of argparse.Namespace); ignored
         @param[in] values       a list of trace levels;
-            each item must be of the form component_name=level
+            each item must be of the form component_name=level or level
+            where level is an integer or a keyword
         @param[in] option_string    option value specified by the user (a str)
         """
+        permittedLevelList = ('DEBUG', 'INFO', 'WARN', 'FATAL')
```

Comment:
Don't need `permittedLevelList`.  If you don't want the below error to say "set(...)", then cast it to a list then.